tests/e2e/scenarios/harvest_nucl_th_and_jlab_curation/RTService/authenticate.yaml

```diff
@@ -0,0 +1,26 @@
+request:
```

Comment:
and all the rt ones too

Reply:
Rt also is different, in the request and the response

Reply:
because we have to send the ticket for jlab linking the editor XD

Reply:
you can match only by the url/method, that allows you to not match by body, and then you don't care :)